### PR TITLE
add import snapshot e2e test

### LINKED ISSUE #317: Add e2e test for snapshot import
**Is your feature request related to a problem? Please describe.**
Add e2e test for snapshot import

### PR DESCRIPTION
**Pull request by AndyXiangLi**
**Is this a bug fix or adding new feature?**
Fixes #317 
**What is this PR about? / Why do we need it?**
Add snapshot import e2e test
**What testing is done?** 
e2e test complete

continue from PR #674

### CODE REVIEW

File: examples/kubernetes/snapshot/specs/snapshot-import/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Prerequisites
+
+1. Kubernetes 1.17+ (CSI 1.0).
+
+1. The `VolumeSnapshotDataSource` must be set in `--feature-gates=` in the `kube-apiserver`. This feature is enabled by default from Kubernetes v1.17+. 
+
+1. Install Snapshot Beta CRDs, Common Snapshot Controller, & CSI Driver (with alpha features) per CSI Snapshotter [Doc](https://github.com/kubernetes-csi/external-snapshotter#usage)
+
+
```
Comment (ayberk) [inline examples/kubernetes/snapshot/specs/snapshot-import/README.md:15]: Do we need these spaces?

File: examples/kubernetes/snapshot/specs/snapshot-import/app/claim.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    - ReadWriteOnce
+  storageClassName: ebs-sc
+  resources:
+    requests:
+      storage: 4Gi
+  dataSource:
+    name: static-snapshot-demo
+    kind: VolumeSnapshot
+    apiGroup: snapshot.storage.k8s.io
+
```
Comment (ayberk) [inline examples/kubernetes/snapshot/specs/snapshot-import/app/claim.yaml:16]: extra line here.

File: examples/kubernetes/snapshot/specs/snapshot-import/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Volume Snapshots
+
+## Overview
+
+This driver implements basic volume snapshotting functionality using the [external snapshotter](https://github.com/kubernetes-csi/external-snapshotter) sidecar and creates snapshots of EBS volumes using the `VolumeSnapshot` custom resources.
+
+## Prerequisites
+
+1. Kubernetes 1.17+ (CSI 1.0).
```
Comment (ayberk) [inline examples/kubernetes/snapshot/specs/snapshot-import/README.md:9]: Why do we need 1.17?

**Review by ayberk — CHANGES_REQUESTED** [review]
Almost there. A couple of minor comments.

File: examples/kubernetes/snapshot/specs/snapshot-import/README.md (hunk 3 above)
Comment (AndyXiangLi) [reply examples/kubernetes/snapshot/specs/snapshot-import/README.md:11]: We are using beta snapshot api, and recording to this article https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-cis-volume-snapshot-beta/, kube version should be 1.17+

File: examples/kubernetes/snapshot/specs/snapshot-import/app/claim.yaml (hunk 2 above)
Comment (AndyXiangLi) [reply examples/kubernetes/snapshot/specs/snapshot-import/app/claim.yaml]: fixed

File: examples/kubernetes/snapshot/specs/snapshot-import/README.md (hunk 1 above)
Comment (AndyXiangLi) [reply examples/kubernetes/snapshot/specs/snapshot-import/README.md]: fixed

File: .github/workflows/helm-chart-release.yaml [hunk 4]
```diff
@@ -22,6 +22,4 @@ jobs:
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.1.0
         env:
-          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (ayberk) [inline .github/workflows/helm-chart-release.yaml:25]: can you rebase master? We shouldn't rollback these changes :)

Comment (AndyXiangLi) [reply .github/workflows/helm-chart-release.yaml]: Done. Sorry forgot to fetch upstream...

**Comment by ayberk** [comment]
/lgtm
/approve

Good job!

**Pull request merged**